userguide/content/en/docs/adding-content/lookandfeel.md

```diff
@@ -6,24 +6,43 @@ description: Customize colors, fonts, code highlighting, and more for your site.
+probably will!) you can very easily override the theme defaults with your own
+project-specific values - Hugo will look in your project files first when
+looking for information to build your site. And because [Docsy uses Bootstrap 5]
+and SCSS for styling, you can override just single values (such as project
+colors and fonts) in its special SCSS project variables file, or do more serious
+customization by creating your own styles.
+
+Docsy also provides options for styling your code blocks, using either Chroma or
+Prism for highlighting.
 
 ## Project style files
 
-To customize your project's look and feel, create your own version of either or both of the following
-Docsy placeholder files (note the **`_project.scss`** suffixes):
-
-- [`assets/scss/`**`_variables_project.scss`**][_variables_project] is where you add project-specific definitions of theme variables such as [site colors](#site-colors), as well as any additional Bootstrap variable values you want to set. You can find a list of Docsy's theme variables and their default values in [<code>assets/scss/<strong>_variables.scss</strong></code>][_variables].  For information about other Bootstrap 4 variables, see [Variable defaults][] and Bootstrap's [v4-dev/scss/_variables.scss][] file.
-- [`assets/scss/`**`_styles_project.scss`**][_styles_project] is where you can add your own custom SCSS styles, including overriding any of the styles in Docsy's theme SCSS files.
+To customize your project's look and feel, create your own version of either or
+both of the following Docsy placeholder files (note the **`_project.scss`**
+suffixes):
+
+- [`assets/scss/`**`_variables_project.scss`**][_variables_project] is where you
+  add project-specific definitions of theme variables such as [site
+  colors](#site-colors), as well as any additional Bootstrap variable values you
+  want to set. You can find a list of Docsy's theme variables and their default
+  values in [`assets/scss/`**`_variables.scss`**][_variables].  For information
+  about other Bootstrap 5 variables, see [Variable defaults] and Bootstrap's
+  [`_variables.scss`] file.
+- [`assets/scss/`**`_styles_project.scss`**][_styles_project] is where you can
+  add your own custom SCSS styles, including overriding any of the styles in
+  Docsy's theme SCSS files.
 
 [_styles_project]: https://github.com/google/docsy/blob/main/assets/scss/_styles_project.scss
 [_variables_project]: https://github.com/google/docsy/blob/main/assets/scss/_variables_project.scss
+[`_variables.scss`]: https://github.com/twbs/bootstrap/blob/v5.2.3/scss/_variables.scss
 [_variables]: https://github.com/google/docsy/blob/main/assets/scss/_variables.scss
-[v4-dev/scss/_variables.scss]: https://github.com/twbs/bootstrap/blob/v4-dev/scss/_variables.scss
+[Docsy uses Bootstrap 5]: /blog/2023/bootstrap-5-migration/
 [Variable defaults]: https://getbootstrap.com/docs/4.1/getting-started/theming/#variable-defaults
```

Comment:
Is that the related 5.3 page? https://getbootstrap.com/docs/5.3/customize/css-variables/

Reply:
Oh thanks! I was clearly zip'n too quickly through this one. I'll fix it soon.

Reply:
Almost, the link we want is https://getbootstrap.com/docs/5.3/customize/sass/#variable-defaults.
Fixed.